mini-modeline.el

```diff
@@ -316,28 +321,33 @@ BODY will be supplied with orig-func and args."
   "Enable `mini-modeline'."
   ;; Hide modeline for terminal, or use empty modeline for GUI.
   (setq-default mini-modeline--orig-mode-line mode-line-format)
-  (setq-default mode-line-format (when (display-graphic-p) '(" ")))
+  (setq-default mode-line-format (when (and mini-modeline-display-gui-line
+                                            (display-graphic-p)) '(" ")))
```

Comment:
Can you put the `'(" ")` in new line to separate from conditional phrase?

Reply:
done